gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.html

```diff
@@ -62,31 +62,39 @@ <h2>{{ endpointGroupData.name }}</h2>
       <!-- Name Column -->
       <ng-container matColumnDef="name">
         <th mat-header-cell *matHeaderCellDef id="name" width="20%">Name</th>
-        <td mat-cell *matCellDef="let element">
+        <td mat-cell [class.endpoint-list__table__backup-row]="element.isBackup" *matCellDef="let element">
```

Comment:
wdyt to add a badge over a color change ? like [secondary] with 
![image](https://user-images.githubusercontent.com/4974420/194893256-86966e9d-d0ad-4ced-84c6-f19e6f32d03a.png)

Reply:
Here it is. You're right it's better like that I think 👍🏼 

![image](https://user-images.githubusercontent.com/25704259/195018482-38f1626e-1a22-4b7d-bde6-24064dbaabdb.png)